sdk/network/Azure.ResourceManager.Network/samples/Generated/Samples/Sample_ExpressRoutePortResource.cs

```diff
@@ -115,7 +115,7 @@ public async Task Update_ExpressRoutePortUpdateTags()
         public async Task GenerateLoa_GenerateExpressRoutePortLOA()
         {
             // Generated from example definition: specification/network/resource-manager/Microsoft.Network/stable/2024-05-01/examples/GenerateExpressRoutePortsLOA.json
-            // this example is just showing the usage of "ExpressRoutePorts_GenerateLoa" operation, for the dependent resources, they will have to be created separately.
+            // this example is just showing the usage of "ExpressRoutePorts_GenerateLOA" operation, for the dependent resources, they will have to be created separately.
```

Comment:
it's `LOA`: https://github.com/Azure/azure-rest-api-specs/blob/49456eb149e3bb5d0528cbca714cfa1806415484/specification/network/resource-manager/Microsoft.Network/stable/2024-05-01/expressRoutePort.json#L561